### DISCUSSION TITLE
Basic questions for adopting Pockethost

### DISCUSSION THREAD

**Discussion by insidesmart** [pockethost/pockethost#311] [Q&A]
Hi,

Pockethost looks nice. Great work. Kudos for your efforts.

I have few questions which I couldnt find answers

- What is the license for this project ? PocketBase is a MIT license project. Can you add a LICENSE.md file for clarity ?
- Since all PB instances are running in docker containers, do you offer ssh (full access/restricted access) for the container ? I have seen FTP access mentioned.

Good luck for 1.0 launch

**Comment by benallfree** [comment]
PocketBase uses the MIT license. Please feel welcome to submit a PR with whatever clarification you desire.

SSH to a Docker container is an interesting idea. What would you do if you had that access? I can think of a few caveats:

1. The Docker container exits when there is no active HTTP traffic
2. The container is ephemeral. Any modifications you made to it would be lost. Your permanent PocketBase files are bound from the host, but those are the same files you can already access via FTP
3. We'd need an SSH tunnel from the origin/host into the Docker container, and it would need to keep the container alive as long as that SSH connection was alive. This could potentially make the container run for a long time.

It's worth investigating further though. Please feel welcome to join the Discord discussion about it https://discord.com/channels/1128192380500193370/1158460622422425620/1159114686306975834

**Comment by insidesmart** [comment]
~~Sorry I was not making it clear. I was highlighting the lack of explicit license mentioned for PocketHost project (in case anyone wants to self host Pockethost in their servers).~~

I just noticed package.json has mentioned the license as MIT. Thanks for that.

Thanks for the thoughtful responses for SSH mode.

- if we want to run any PocketBase cli commands like ```pocketbase admin update```,  I thought having a SSH access would help. Is there an alternate way to use PocketBase cli commands ?

- I was actually planning to host a SSG app (similar to PB Admin app) as part of PocketBase runtime. I thought SSH access would help there. It looks like you have already enabled this usecase via FTP.

I will checkout discord soon.

Once again, Nice work in putting together a multi tenant platform single handedly.

**Reply by benallfree** [reply]
> single handedly

https://github.com/pockethost/pockethost/graphs/contributors :)
